packages/hardhat-core/test/internal/hardhat-network/provider/node.ts

```diff
@@ -5,8 +5,10 @@ import VM from "@ethereumjs/vm";
 import { AfterBlockEvent, RunBlockOpts } from "@ethereumjs/vm/dist/runBlock";
 import { assert } from "chai";
 import { Address, BN, bufferToHex } from "ethereumjs-util";
+import { pass } from "fp-ts/lib/Writer";
```

Comment:
This doesn't seem to be used.

Reply:
Vscode might pulled a little sneaky on me. Fixed